render.ts

```diff
@@ -11,13 +11,30 @@ import renderToString from 'https://esm.sh/preact-render-to-string@6.1.0?deps=pr
 import { VNode } from 'https://esm.sh/preact@10.17.1?target=es2022'
 import { Context } from './mod.ts'
 
+const HEAD_TAG = '<head>'
+
 export function render(c: Context, Component: VNode) {
   const htmlString = renderToString(Component)
 
   try {
     const { html, css } = extract(htmlString)
-
-    c.res.body = `<style>${css}</style><body>${html}</body>`
+    console.log(css.length)
+    const startIdxOfHeadContent = html.indexOf(HEAD_TAG)
+    const endIdxOfHeadContent = html.indexOf(
+      `${HEAD_TAG.at(0)}/${HEAD_TAG.slice(1)}`,
+    )
+    const headContent =
```

Comment:
maybe using regex would be more efficient?

```suggestion
    const headRegex = /<head>([\s\S]*?)<\/head>/g;

    const headContent = headRegex.match(html)[1];
```

Reply:
regex is genuinely slower, no?

Reply:
![image](https://github.com/cheetahstudio/cheetah/assets/47074495/62b86b60-b08d-4a3a-b2a8-8fd110e1c3e1)

LOL i never realized regex is that slow

this is all good then 😄

Reply:
59 ops/s??? oh my! I heard it was slow... but didn't think it would be THAT slow. lol